internal/person/service.go

```diff
@@ -21,7 +22,7 @@ func (p *personService) GetProfile(ctx context.Context, iamID string) (Person, e
 	var personObj Person
 	person, err := p.repo.GetProfile(ctx, iamID)
 	if err != nil {
-		return Person{}, nil
+		return Person{}, errors.New("profile_not_found")
```

Comment:
can revert - just need to let the error from `p.repo.GetProfile(ctx, iamID)` pass through :)

Reply:
Updated